exemplar/src/main/java/com/squareup/exemplar/ExemplarJavaConfigModule.java

```diff
@@ -4,7 +4,13 @@
 import misk.config.ConfigModule;
 
 public class ExemplarJavaConfigModule extends AbstractModule {
```

Comment:
would it be simpler to just inline this entire module at the callsite?

Reply:
I like that. Will fix up